application_default.properties

```diff
@@ -0,0 +1,69 @@
+# Default demo application settings
```

Comment:
I assume if the user try to use "application_default.properties", they'll need to rename this file when copy over to config folder. 
I think we should just use "application.properties" as the name of the file and I assume this won't affect the default behavior to read the original properties file when running from source right?

Reply:
Yes, you're right. Let's just use  "application.properties".